src/native/libs/build-native.proj

```diff
@@ -26,6 +26,11 @@
     <ProjectReference Include="$(LibrariesProjectRoot)native-binplace.proj" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetOS)' == 'wasi'">
+    <WASI_SDK_PATH Condition="'$(WASI_SDK_PATH)' == '' or !Exists('$(WASI_SDK_PATH)/VERSION')">$([MSBuild]::NormalizeDirectory($(RepoRoot), 'src', 'mono', 'wasi', 'wasi-sdk'))</WASI_SDK_PATH>
```

Comment:
This does not propagate to the linux shell for me and uses original `WASI_SDK_PATH` env variable set to too old SDK. I need to improve this.

Reply:
solved by adding `_BuildNativeEnvironmentVariables`